cmd/notation/registry.go

```diff
@@ -57,17 +56,18 @@ func getRepository(ctx context.Context, inputType inputType, reference string, o
 }
 
 // getRemoteRepository returns a registry.Repository.
-// When experimental feature is disabled OR allowReferrersAPI is not set,
-// Notation always uses referrers tag schema to store and consume signatures
-// by default.
-// When experimental feature is enabled AND allowReferrersAPI is set, Notation
-// tries the Referrers API, if not supported, fallback to use the Referrers
-// tag schema.
+// When forceReferrersTag is set, Notation will always generate an image index
```

Comment:
for this function, we always need to set `forceReferrersTag`, so is it better to be:
```suggestion
// When forceReferrersTag is true, Notation will always generate an image index
```